src/box/relay.cc

```diff
@@ -1119,6 +1119,14 @@ static void
 relay_send(struct relay *relay, struct xrow_header *packet)
 {
 	ERROR_INJECT_YIELD(ERRINJ_RELAY_SEND_DELAY);
+	/**
```

Comment:
This is the least crutchy solution I managed to find. The solution for initial https://github.com/tarantool/tarantool/issues/10089 is easy, we can just patch `relay_send_initial_join_row` to make it send raft requests, but I don't want to send `GROUP_LOCAL` rows at all, this seems incorrect ideologically.

It would be perfect, if sending raft state from `box_process_subscribe` could be done via `relay_send`, but I didn't manage to make it work, seems like it's there for the reason.

Reply:
The label `2.11` to #10089 suggests that it should be backported to 2.11 so we need to take it into account when checking `version_id`. Also, since the version check is used in two places, I'd put it in a helper function.

Reply:
Resolving since the issue is addressed